Dockerfile

```diff
@@ -1,27 +1,26 @@
 FROM node:14-alpine AS BUILD
-COPY . /src
 
 # git is needed to install Half-Shot/slackdown
 RUN apk add git
 WORKDIR /src
+
+COPY package.json package-lock.json /src/
 RUN npm install
+COPY . /src
```

Comment:
If we do this anyway, what's the advantage of two copy operations?

Reply:
The advantage is that I first only copy the package.json and package-lock.json files. So if they didn't change since the last build docker can use the cache for this step, hence it can also use the cache for the `RUN npm install` step.